canonical-web-and-design/charmhub.io
# Small fixes

## Done

-Make the spinner spin
- Replace karlos with juju logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- The homepage should have the spinner spinning at the bottom of the page
- Karlos should be replaced with Juju logo


## Issue / Card

Fixes #

## Screenshots

[if relevant, include a screenshot]
